cumulusci/tasks/marketing_cloud/api.py

```diff
@@ -0,0 +1,205 @@
+import requests
+from lxml import etree
+
+from cumulusci.salesforce_api import mc_soap_envelopes as envelopes
+
+from .base import BaseMarketingCloudTask
+
+
+class MarketingCloudCreateSubscriberAttribute(BaseMarketingCloudTask):
```

Comment:
I'd be inclined to remove MarketingCloud from the names of these classes; they are already in a Python package named marketing_cloud